### ISSUE TITLE
Package request: bizhawk

### ISSUE THREAD

**Issue by keenanweaver** [NixOS/nixpkgs#301131]
**Project description**

BizHawk is a multi-platform emulator with full rerecording support and [Lua scripting](https://tasvideos.org/Bizhawk/LuaFunctions). BizHawk focuses on core accuracy and power user tools while still being an easy-to-use emulator for casual gaming.

**Metadata**

* homepage URL: https://tasvideos.org/Bizhawk
* source URL: https://github.com/TASEmulators/BizHawk
* license: MIT
* platforms: linux

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by WillemToorenburgh** [comment]
As a relative Nix/NixOS noob, I'd be interested in giving this a shot. Interestingly, the one(?) maintainer for the project who does Linux support has also put in Nix files: a [`default.nix`](https://github.com/TASEmulators/BizHawk/blob/master/default.nix) and a [`shell.nix`](https://github.com/TASEmulators/BizHawk/blob/master/shell.nix). If there's any known guidance on how to use existing Nix files as a basis for a NixPkgs submission, I'd very much appreciate it! I'll do some searching in the meantime.

**Comment by WillemToorenburgh** [comment]
Turns out they've already got some pretty extensive Nix usage in the repo! Of particular note is [`Dist/packages.nix`](https://github.com/TASEmulators/BizHawk/blob/master/Dist/packages.nix), which looks to be a pretty complete derivation definition. I wonder if just wholesale copying that file would be enough for NixPkgs... I feel like doing so would just be stealing/misappropriating @YoshiRulz 's fine work.

**Comment by YoshiRulz** [comment]
Not at all! You're welcome to re-use any of that for inclusion in Nixpkgs. I haven't done it because I don't think it would fit the contribution guidelines.

And as you say, we do already have a working setup checked-in to our repo. Anyone watching this thread who just wants to run EmuHawk can go to https://github.com/TASEmulators/BizHawk#nixnixos to get started.
